LANGUAGE.md

```diff
@@ -31,7 +31,7 @@ Additionally, the following symbols should not be used as *Identifiers*:
 Single line comments are started with `shh` and ended on a new line:
 ```dogescript
 shh This is a single line comment
-shh more single line 
+shh more single line
```

Comment:
This seems like an irrelevant change?

Reply:
It seems there was a whitespace char: `" "` in there and my atom removed it.